changelog/@unreleased/pr-470.v1.yaml

```
@@ -0,0 +1,6 @@
+type: improvement
+improvement:
+  description: Add ability to do multi platform docker builds using buildx
+  links:
+    - https://github.com/palantir/gradle-docker/pull/413
```

```suggestion
    - https://github.com/palantir/gradle-docker/pull/527
```

Can we also rename the changelog file to `changelog/@unreleased/pr-527.v1.yaml`? That way we keep the link of what PR merged in the changes.